beanstalk_worker/worker_start_scan.py

```diff
@@ -99,20 +105,45 @@ def run_a_scanner(self, scanner, image_under_test):
                 msg="Finished running %s scanner." % scanner
             )
 
-        # if scanner failed to run, this will return {}, do check at receiver end
+        # if scanner failed to run, this will return {}, do check at receiver
+        # end
         return json_data
 
+    def export_scanners_status(self, status, status_file_path):
+        """
+        Export status of scanners execution for build in process
+        """
+        try:
+            fin = open(status_file_path, "w")
+            json.dump(fin, status)
+        except IOError as e:
+            logger.log(
+                level=logging.CRITICAL,
+                msg="Failed to write scanners status on NFS share."
+            )
+            logger.log(
+                level=logging.CRITICAL,
+                msg=str(e)
+            )
+        else:
+            logger.log(
+                level=logging.INFO,
+                msg="Wrote scanners status to file: %s" % status_file_path
+            )
+        finally:
+            return status_file_path
```

Comment:
Same feedback as `export_linter_status` function. The returned value doesn't seem to be getting used anywhere.